src/BrazeKit-dev.js

```diff
@@ -571,6 +572,26 @@ var constructor = function () {
         return reportEvent;
     }
 
+    function setSubscriptionGroups(key, value) {
+        const subscriptionGroupId = parsedSubscriptionGroupMapping[key];
+        if (!(typeof value === 'boolean')) {
+            kitLogger("Can't call setSubscriptionGroups on forwarder " +
+	            name +
+	            ', setSubscriptionGroups must set this value to a boolean');
+            return;
+        } else {
+            if (!value) {
+                kitLogger('braze.getUser().removeFromSubscriptionGroup', subscriptionGroupId);
+
+	            braze.getUser().removeFromSubscriptionGroup(subscriptionGroupId);
+            } else {
+                kitLogger('braze.getUser().addToSubscriptionGroup', subscriptionGroupId);
+
+	            braze.getUser().addToSubscriptionGroup(subscriptionGroupId);
+            }
+        }
+    }
```

Comment:
I think the nested if/else statements makes this function a bit hard to read. 

Also `typeof value !== 'boolean'` reads better than `!(typeof value === 'boolean')`.

P.S. Nit: Set up your linter!

```suggestion
    function setSubscriptionGroups(key, value) {
        var subscriptionGroupId = parsedSubscriptionGroupMapping[key];

        if (typeof value !== 'boolean') {
            kitLogger("Can't call setSubscriptionGroups on forwarder " +
                name +
                ', setSubscriptionGroups must set this value to a boolean');
            return;
        }

        var action = value ? 'addToSubscriptionGroup' : 'removeFromSubscriptionGroup';
        kitLogger('braze.getUser().' + action, subscriptionGroupId);
        braze.getUser()[action](subscriptionGroupId);
    }
```